features/admin.authentication.v1/hooks/use-sign-in.ts

```diff
@@ -125,38 +125,74 @@ const useSignIn = (): UseSignInInterface => {
      * @param orgType - Type of the organization. Ex: sub organization, etc.
      */
     const setCustomServerHost = (orgType: string): void => {
-        // In case of failure customServerHost is set to the serverHost.
-        let customServerHost: string = Config?.getDeploymentConfig()?.serverHost;
+        const disabledFeatures: string[] = window["AppUtils"]?.getConfig()?.ui?.features?.branding?.disabledFeatures;
+
+        if (!disabledFeatures?.includes("branding.hostnameUrlBranding")) {
+            axios
+                .get(Config.getServiceResourceEndpoints().wellKnown)
```

Comment:
Isn't there a hook for this GET operation already defined?

Reply:
AFAIK, there's no such reusable hook.
Lets track this separately to create a custom hook (ex: `useOIDCDiscovery`) and to use the `httpRequest` from the SDK with `attachToken=false` instead of using `axios` raw.

@Yasasr1

Reply:
Ack will create an issue

Reply:
https://github.com/wso2/product-is/issues/20472